src/lib.rs

```diff
@@ -0,0 +1,89 @@
+pub mod read_create {
+    extern crate calamine;
+    use calamine::{open_workbook, Xlsx};
+    use std::fs::File;
+    use std::io::BufReader;
+    pub fn create_file(input_file: &str) -> File {
+        let file_handle = File::create(input_file).expect("unable to create output file");
+        file_handle
+    }
+    pub fn open_file(output_file: &str) -> Xlsx<BufReader<File>> {
+        let workbook: Xlsx<_> = open_workbook(output_file).expect("unable to open the input file");
+        workbook
+    }
+}
+
+pub mod operation {
+    use std::fs::File;
+    use std::io::BufReader;
+    use std::io::Write;
+    use std::string::String;
+    extern crate calamine;
+    use calamine::{Reader, Xlsx};
```

Comment:
@harishym now your code looks good and easy to read. But one thing why u need to define these use modules for separate function???? this is not a good programing practice, It will make your code lengthy also. Please declare all use modules at the beginning of the module so that u don't need to write it again and again

Reply:
i have defined use modules at the beginning of the each module. you mean to say declare all the use modules for all the modules only once?

Reply:
if i declare use modules outside the modules, it is asking me to include those use modules inside the modules defined by me.

Reply:
not outside the module I was taling abount the functions